frontend/app/forms/inputs/text-input-group.component.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { useCss } from "kremling";
+import { DataContainerContext } from "../data-container.component";
+
+export default function TextInputGroup(props) {
+  const dataContext = React.useContext(DataContainerContext);
+  var inputs = Object.assign({}, ...props.inputs);
```

Comment:
Do we need this default value? To me it doesn't make sense that someone would use TextInputGroup without passing in the inputs.

Reply:
From my understanding this is creating a new object. Which is a group of inputs that you can create  more groups from.